ui-frontend/packages/catalog-ui-search/src/main/webapp/index.html

```diff
@@ -29,19 +29,98 @@
 </head>
 
 <body>
-    <header></header>
+    <header>
+        <style type=text/css>
+            #incompatible-browser-bg {
+                position: fixed;
+                top: 0;
+                left: 0;
+                text-align: center;
+                line-height: 30px;
+                width: 100%;
+                height: 100%;
+                background-color: #0b1821;
+                overflow: auto;
+                font-family: 'Open Sans', arial, sans-serif;
+            }
+
+            #incompatible-browser {
+                display: table-cell;
+                vertical-align: middle;
+                padding: 20px;
+            }
+
+            #incompatible-browser li {
+                width: 176px;
+                display: inline-block;
+                vertical-align: top;
+                margin: 10px;
+                zoom: 1;
+                *display: inline;
+            }
+        </style>
+    </header>
     <!--[if lt IE 10]>
-<div id="iewarning" class="iewarning">
-    <div style="color:yellow">
-        Internet Explorer 9 and below are not supported. Please upgrade your browser or use the
-        latest version of Firefox, Chrome, or Safari.
+    <div id="incompatible-browser-bg" style="color: white;">
+      <div style="display: table; width: 100%; height: 100%;">
+        <div id="incompatible-browser">
+          <h1 style="font-size: 20px; margin-bottom: 20px;">Browser Unsupported</h1>
+          <div class="browser">
+            <img src="./img/no-ie.png" alt="no-ie" style="margin-right: 24px;" />
+            <br />
+            <label style="font-size: 12px;">IE (Internet Explorer)</label>
+          </div>
+          <p style="font-size: 16px;">
+            Please use one of the following supported browsers to access the
+            system.
+          </p>
+          <ul id="supported-browsers" style="list-style-type:none; padding: 0px">
+            <li class="browser">
+              <img src="./img/edge.png" alt="edge" style="margin-right: 24px;" />
+              <label style="font-size: 12px;">Microsoft Edge</label>
+            </li>
+            <li class="browser">
+              <img src="./img/firefox.png" alt="firefox" style="margin-right: 24px;" />
+              <label style="font-size: 12px;">Mozilla Firefox</label>
+            </li>
+            <li class="browser">
+              <img src="./img/chrome.png" alt="chrome" style="margin-right: 24px;" />
+              <label style="font-size: 12px;">Google Chrome</label>
+            </li>
+          </ul>
+        </div>
+      </div>
```

Comment:
This whole section above is pretty much duplicate code of the section below. 😅
The only reason I put this here is because IE 9 and below completely bypass the Javascript, so I can't rely on browser detection to change the `display` from `none` to `block` and I couldn't think of a better way to do it. Please let me know if you have any suggestions!